pages/guideforms.py

```diff
@@ -71,6 +100,24 @@ def validate(self, value):
     'pattern': EMAIL_ADDRESSES_REGEX
 }
 
+# From https://rodneyrehm.de/t/url-regex.html#imme_emosol+ht-%26f-tp%28s%29
+URL_REGEX = '[ ]*(https?)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?[ ]*'
```

Comment:
I don't see why we need the `((-\.)?` part.

And, I think the domain part could be more clearly separate from the path-query-anchor part.  E.g., there should not be a `#` in the domain part.

Let's try:
```
URL_REGEX = '[ ]*((https?)://)?([\w-]+(\.[\w-]+)+)(/[^\s]*)?[ ]*'
```

Reply:
Added comment about using the imme_emosol pattern.

I don't understand how that pattern works, but testing e.g. http;//-www.com or http://.www.com, it actually correctly fails to match.  See http://regexr.com for a handy regex tester.

So I think we should leave this as is, for now anyway.